packages/mongo-schema/schema_validate.js

```
@@ -0,0 +1,284 @@
+      const itemKey = `${key}.${i}`;
+      validateField(ir, itemKey, itemDesc, rootDoc, value[i], errors, options);
+    }
+  }
+}
+
+function validateConstraints(key, desc, value, errors, options) {
+  const label = desc.label || key;
+
+  // denyInsert / denyUpdate
+  if (desc.denyInsert && options.isInsert && value !== undefined) {
+    errors.push({ name: key, type: ErrorTypes.DENY_INSERT, value, message: `${label} cannot be set during insert` });
```

ESLint

[error] Replace `·name:·key,·type:·ErrorTypes.DENY_INSERT,·value,·message:·`${label}·cannot·be·set·during·insert`` with `⏎······name:·key,⏎······type:·ErrorTypes.DENY_INSERT,⏎······value,⏎······message:·`${label}·cannot·be·set·during·insert`,⏎···`

(prettier/prettier)